carsons/carsons.py

```diff
@@ -265,3 +275,39 @@ def GMR_cn(self, phase) -> float:
         k = self.neutral_strand_count[phase]
         R = self.radius[phase]
         return (GMR_s * k * R**(k-1))**(1/k)
+
+
+class MultiConductorCarsonsEquations(CarsonsEquations):
+    def __init__(self, model):
+        super().__init__(model)
+        self.radius: Dict[str, float] = model.radius
+        self.secondary: bool = model.secondary
+        self.insulation_thickness: Dict[str, float] = \
+            model.insulation_thickness
+
+    def compute_d(self, i, j) -> float:
+        return (self.radius[i] + self.radius[j]
```

Comment:
It might be a good idea to add a note here about the assumption we are making, which is that 

1) we assume all the conductors are touching eachother and are thus equidistant
2) in the case we have 4 conductors, we are neglecting that there may be space across the bundle

![image](https://user-images.githubusercontent.com/2414360/74180943-5c4c9f80-4c0e-11ea-9ca0-f76da2e688a6.png)

Reply:
right yeah i'll add a note